packages/loopring_v3/contracts/core/impl/ExchangeV3.sol

```diff
@@ -105,34 +105,6 @@ contract ExchangeV3 is IExchangeV3, ReentrancyGuard
         );
     }
 
-    function cloneExchange(
```

Comment:
So we're going to put the exchange behind a proxy somewhere else?

Reply:
I will deploy the proxy contract manually, there is no need to put the code in the contract.